recipes/seq2c/meta.yaml

```diff
@@ -1,14 +1,15 @@
 package:
   name: seq2c
-  version: '2019.05.30'
+  version: '1.2'
 
 source:
-  url: https://github.com/AstraZeneca-NGS/Seq2C/archive/1603613.tar.gz
-  sha256: 673deeffd4364f45e798245dee1bbd61211dcdea36eb2a94e13a65ba139ee870
+  url: https://github.com/AstraZeneca-NGS/Seq2C/archive/v1.2.tar.gz
+  sha256: 4b24f81eaeaec709f9b7564d8f5ca33560e905ce4f78eedf5e86165369e13d62
 
 build:
   number: 0
   skip: False
+  noarch: generic
 
 requirements:
   build:
```

Comment:
In your case, since you are not building anything, just having the `run` requirements section should be sufficient I think.